packages/wrangler/package.json

```diff
@@ -60,7 +60,7 @@
 		"emit-types": "tsc -p tsconfig.emit.json && node -r esbuild-register scripts/emit-types.ts",
 		"prepublishOnly": "SOURCEMAPS=false npm run build",
 		"start": "npm run bundle && cross-env NODE_OPTIONS=--enable-source-maps ./bin/wrangler.js",
-		"test": "npm run assert-git-version && jest",
+		"test": "npm run assert-git-version && cross-env TZ=UTC jest",
```

Comment:
@petebacondarwin will this be an issue? Just wanted to ensure there's no difference between tests running in CI and on dev machines

Reply:
Hmm, is it to avoid accidentally crossing date boundaries in tests? Is it possible to make the test more deterministic (i.e. use a mock clock instead of real clock)?

Reply:
The problem is that this PR introduces a function that throws an error with a helpful suggestion of the format a timestamp could be in, by converting `new Date()` to an ISO timestamp using the user's local timezone:
```
"Invalid timestamp 'asdf'. Please provide a valid Unix timestamp or ISO string, for example: 2023-08-01T00:00:00.000+00:00
For accepted format, see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#date_time_string_format"
```

https://github.com/cloudflare/workers-sdk/pull/3579/files#diff-b8a843f61d481f1d39b7595d6a48be61cf00ae092f494700f99eb30561d9eaa0R60-R67